packages/babel-parser/src/plugins/typescript/index.ts

```diff
@@ -2350,49 +2353,7 @@ export default (superClass: ClassWithMixin<typeof Parser, IJSXParserMixin>) =>
       );
     }
 
-    tsParseExpressionStatement(
-      node: Undone<N.TsModuleDeclaration>,
-      expr: N.Identifier,
-      decorators: N.Decorator[] | null,
-    ): N.Declaration | undefined {
-      switch (expr.name) {
-        case "declare": {
-          const declaration = this.tsTryParseDeclare(node);
-          if (declaration) {
-            declaration.declare = true;
-          }
-          return declaration;
-        }
-        case "global":
-          // `global { }` (with no `declare`) may appear inside an ambient module declaration.
-          // Would like to use tsParseAmbientExternalModuleDeclaration here, but already ran past "global".
```

Comment:
This todo item is also resolved now that the global module parsing has been moved to `parseStatementContent` when `global` is not yet consumed.